chipy_org/templates/flatpages/default.html

```diff
@@ -1,7 +1,24 @@
 {% extends "shiny/slim.html" %}
-{% block page_header %}{{ flatpage.title | upper }}{% endblock %}
+
```

Comment:
Please remove all the changes related to the flatpages and move them to a different PR as discussed.